### PR TITLE
Add roles to the collection

### PR DESCRIPTION
**Pull request by outcatcher**
Copy roles from `csm-ansile-roles`

Add zuul integration

### CODE REVIEW

File: roles/build_infrastructure/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+galaxy_info:
+  role_name: build_infrastructure
+  min_ansible_version: 2.8
+  description: "installing terraform on the remote host"
```
Comment (anton-sidelnikov) [inline roles/build_infrastructure/meta/main.yml]: build terraform scripts?

File: roles/build_infrastructure/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+tmp_dir: '/tmp'
```
Comment (lego963) [inline roles/build_infrastructure/defaults/main.yml]: doueble quotes pls

File: roles/build_infrastructure/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+key_name: null
+terraform_base_dir: "infrastructure"
+
+username: null
+password: null
+
+desired_state: present
+terraform_workspace: 'test'
```
Comment (lego963) [inline roles/build_infrastructure/defaults/main.yml]: and here

File: roles/build_infrastructure/meta/main.yml (hunk 1 above)
Comment (anton-sidelnikov) [reply roles/build_infrastructure/meta/main.yml]: Strange description

File: roles/build_infrastructure/molecule/default/cleanup.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+- name: Cleanup
+  hosts: instance
+  vars:
+    requirements: "requirements.txt"
+    scenario_name: "test"
+    key_name: "key_scenario1"
```
Comment (lego963) [inline roles/build_infrastructure/molecule/default/cleanup.yml:7]: we dont have `scenario*`
Now it is called `*_monitoring`

File: roles/build_infrastructure/molecule/default/converge.yml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+- name: Converge
+  hosts: instance
+  vars:
+    requirements: "requirements.txt"
+    scenario_name: "test"
+    key_name: "key_scenario1"
```
Comment (lego963) [inline roles/build_infrastructure/molecule/default/converge.yml:7]: plus one

File: roles/build_infrastructure/tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+- name: Terraform pre_build
+  include_tasks: pre_build.yml
+
+- name: Terraform build
+  include_tasks: build.yml
+  when: infra_state == 'present'
```
Comment (lego963) [inline roles/build_infrastructure/tasks/main.yml]: double quotes pls

**Review by lego963 — CHANGES_REQUESTED** [review]
Can you please make double quotes in `*.yaml` and single quotes in `*.py`?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: roles/build_infrastructure/molecule/default/cleanup.yml (hunk 4 above)
Comment (outcatcher) [reply roles/build_infrastructure/molecule/default/cleanup.yml:7]: This is test cleanup, this has nothing with the CSM naming convention.

File: roles/build_infrastructure/molecule/default/converge.yml (hunk 5 above)
Comment (outcatcher) [reply roles/build_infrastructure/molecule/default/converge.yml:7]: See previous

**Review by lego963 — DISMISSED** [review]

**Review by lego963 — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Review by Polina-Gubina — APPROVED** [review]

**Pull request merged**